us.zoom.Zoom.appdata.xml

```diff
@@ -24,6 +24,7 @@
     </screenshot>
   </screenshots>
   <releases>
+    <release version="3.0.285090.0826" date="2019-08-27"/>
```

Comment:
You might reasonably ask whether it is straightforward to match the attribute order on the other `<release>` elements, to which the answer is “nothing is straightforward with SAX”. (There's an xfail test in my checker branch for this.)